### PR TITLE
[tests-only] Adds version page object for e2e tests

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
This PR adds version page object for e2e tests

## Related Issue

- Part of https://github.com/owncloud/web/issues/6344

## Motivation and Context
There was no separate version page, this PR makes new version page which makes easier to organize and manage code.

## How Has This Been Tested?

- locally
- ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] E2E tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/e2e/support/page/files/version.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Actor } from '../../types'
+import { filesCta } from '../../cta'
+
+export class VersionPage {
+  private readonly actor: Actor
+  private readonly fileVersionSidebarXPath: string
```
Comment (ScharfViktor) [inline tests/e2e/support/page/files/version.ts:6]: I like it, but I think it's the Locator, not the string. I would suggest to do like here: https://playwright.dev/docs/test-pom

File: tests/e2e/support/page/files/version.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  constructor({ actor }: { actor: Actor }) {
+    this.actor = actor
+    this.fileVersionSidebarXPath = '//*[@id="oc-file-versions-sidebar"]/table/tbody/tr'
```
Comment (ScharfViktor) [inline tests/e2e/support/page/files/version.ts:10]: and here we can do something like: `this.fileVersionSidebarXPath = page.locator('//*[@id="oc-file-versions-sidebar"]/table/tbody/tr')`

**Review by ScharfViktor — CHANGES_REQUESTED** [review]

**Review by ScharfViktor — APPROVED** [review]

**Comment by ScharfViktor** [comment]
looks good to me. Thanks for the input. Perhaps @pascalwengerter can comment on something

**Review by SwikritiT — COMMENTED** [review]
LGTM

**Review by phil-davis — APPROVED** [review]

**Pull request merged**